docs/source/index.rst

```diff
@@ -11,69 +11,18 @@ JupyterLab is the next-generation web-based user interface for Project Jupyter.
 
 .. image:: ./images/jupyterlab.png
    :align: center
-   :class: jp-screenshot
+   :class: jp-screenshot shadow
 
+Get started
+===========
```

Comment:
I would drop this title as this is a table of content and no really a how to get started with JupyterLab.

```suggestion
```